meson_options.txt

```diff
@@ -8,7 +8,8 @@ option('enable-uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('enable-uefi-labels', type : 'boolean', value : true, description : 'enable UEFI labels support')
 option('enable-dell', type : 'boolean', value : true, description : 'enable Dell-specific support')
 option('enable-amt', type : 'boolean', value : true, description : 'enable Intel AMT support')
-option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable Thunderbolt support')
+option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable legacy Thunderbolt support')
```

Comment:
@ybernat it is enabled by default. The tests are optional and are built if `umockdev` is present.